content/version/2/0/code_of_conduct.fr.md

```diff
@@ -0,0 +1,132 @@
+ou l'identité et l'orientation sexuelle.
+
+Nous nous engageons à agir et interagir de manière à contribuer à une communauté 
+ouverte, accueillante, diverse, inclusive et saine.
+
+## Nos critères
+
+Exemples de comportements qui contribuent à créer un environnement positif :
+
+* Faire preuve d'empathie et de bienvaillance envers les autres membres
+* Le respect des différents points de vue, opinions et expériences vécues
+* Donner et accepter poliment les critiques constructives
+* Assumer ses responsabilités et s'excuser auprès des personnes affectées par nos erreurs et apprendre de ces expériences
+* Se concentrer sur ce qui est le meilleur non pas uniquement pour nous en tant qu'individu, mais pour la communauté
+
+Exemples de comportements non acceptables :
+
+* L'utilisation de langage ou d'imageries sexualisés et les avances sexuelles non
+  sollicitées
+* Le _trolling_, les commentaires insultants ou désobligeants et les attaques
+  personnelles ou d'ordre politique
+* Le harcèlement en public ou en privé
+* La publication d'informations privées de tierces personnes, telles que des
+  adresses physiques ou électroniques, sans permission explicite
+* Toute conduite qui pourrait être raisonnablement considérée comme inappropriée
+  dans le milieu professionnel
+
+## Nos responsabilités
+
+Les dirigeants de communauté sont chargés de clarifier et de faire respecter nos normes de
+comportements acceptables et prendront des mesures correctives appropriées et équitables comme
+réponse à tout comportement qu'ils jugent inapproprié, menaçant, offensant ou nuisible.
+
+Les dirigeants de communauté ont le droit et la responsabilité de supprimer, modifier ou rejeter
+les commentaires, les _commits_, le code, les modifications du wiki, les questions et autres contributions qui
+ne sont pas alignés sur ce Code de Conduite et communiqueront les raisons de la modération
+le cas échéant.
+
+## Portée d'application
+
+Ce Code de Conduite s'applique à la fois au sein des espaces du projet ainsi que
+dans les espaces publics lorsqu'un individu représente officiellement le projet ou sa
+communauté. Font parties des exemples de représentation d'un projet ou d'une
+communauté le fait d'utiliser une adresse email propre au projet, de poster sur
+les réseaux sociaux avec un compte officiel ou d'intervenir pour représenter le
+projet au cours d'un évènement en ligne ou hors-ligne.
+
+## Application
+
+Les cas de comportements abusifs, harcelants ou tout autre comportement
+inacceptables peuvent être signalés en contactant l'équipe du projet à
+[INSÉRER UNE ADRESSE EMAIL]. 
+Toutes les plaintes seront examinées et feront l'objet d'une enquête rapide et équitable.
+
+Tous les dirigeants de communauté sont tenus de respecter la vie privée et la sécurité des
+personnes ayant signalées un incident.
+
+## Directives d'application
+
+Les dirigeants de communauté suivront ces lignes directrices sur l'Impact Communautaire afin de déterminer

```

Comment:
We could keep [leader](https://www.larousse.fr/dictionnaires/francais/leader/46518?q=Leader#46441) here:
```suggestion
Les leaders de la communauté suivront ces lignes directrices sur l'Impact Communautaire afin de déterminer

```

Reply:
I agree "dirigeants" sounds a bit stiff and old-fashioned, but "leaders" has different but equally negative connotions, so my inclination would be towards the former, which isn't an imported neologism.

Reply:
Since we were talking about inclusive language and as tobie mentionned in one of his suggestions that we could use "dirigeant•e•s" instead of "dirigeants" or even "leader". This seems like a better idea to me (at least, if we decide to move towards an inclusive language).